_layouts/base.html

```diff
@@ -151,6 +151,7 @@ <h1><a href="{{ site.baseurl }}/">{{ site.title }}</a></h1>
         ));
       };
     </script>
+    <script src="{{ "/assets/js/copy.js" | relative_url }}" async></script>
```

Comment:
@MikeMcQuaid will this load properly in formulae ?

Otherwise I could add the js to the script tag above, and avoid a separate js request.

Reply:
> Otherwise I could add the js to the script tag above, and avoid a separate js request.

Yeh, good idea 👍🏻. You'll want to make the JS a function called by something like `<body onload="setupCopyables()">`

Reply:
Done!